src/App.js

```diff
@@ -54,7 +54,7 @@ const App = () => {
               </span>
             )}
           </p>
-        </div>
+        </div> */}
```

Comment:
I wonder if we'll still need this code that's been commented out. Maybe we can delete it by now?